sklearn/decomposition/online_lda.py

```diff
@@ -19,9 +19,8 @@
 from ..utils import (check_random_state, check_array,
                      gen_batches, gen_even_slices)
 from ..utils.fixes import logsumexp
-from ..utils.validation import check_non_negative
+from ..utils.validation import check_non_negative, check_is_fitted
```

Comment:
```suggestion
from ..utils.validation import check_non_negative
from ..utils.validation import check_is_fitted
```

Reply:
We don't have a real convention for that.

My personal thoughts on that are the following:

* import that we are almost sure to not modify can go on a single line (this is the case with joblib)
* imports that are more probable to be changed (this is the case with validation --- we might add or remove an import in the future) could go on different lines. On a single line, we can get more diff just for reordering import and line breaking.

But I agree that this is only my personal taste. Both are valid and I will not block the merge for that. Choose what you prefer.

Reply:
Agreed -- will change.